<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchWidget</class>
 <widget class="QWidget" name="SearchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>30</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Find:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="input"/>
   </item>
   <item>
    <widget class="QToolButton" name="findPrevButton">
     <property name="toolTip">
      <string>Press to find the previous occurrence</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="findNextButton">
     <property name="toolTip">
      <string>Press to find the next occurrence</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="caseCheckBox">
     <property name="text">
      <string>Match case</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="wrapCheckBox">
     <property name="text">
      <string>Wrap around</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="infoLabel">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="closeButton">
     <property name="toolTip">
      <string>Press to close the window</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>caseCheckBox</tabstop>
  <tabstop>wrapCheckBox</tabstop>
  <tabstop>findNextButton</tabstop>
  <tabstop>findPrevButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
